__test__/arraySorting.test.js

```diff
@@ -1,7 +1,7 @@
 const { expect } = require("expect");
 const Printname = require("../lib/arraySorting");
 
-const names = ["Alice", "Bob", "Jeremy", "Sam", "Henry", "Sarah", "Ashely"];
+const names = "Alice";
 
 test("check to see if the name prints out", () => {
   expect(Printname()).toContain("Hello my name is Alice");
```

Comment:
You could just pass this `names` as an argument to Printname() like so ~

`expect(Printname(names)).toContain("Hello my name is Alice");`

Reply:
You can add back your full array of names here. But your test will be different. You probably want something like this.

```
Printname(names).forEach((name,index) => {
    expect(name).toContain(`Hello my name is ${names[index]}`);
})
```